.github/build.in.yml

```
@@ -63,6 +63,18 @@ jobs:
         run: |
           lake exe lint-style
 
+      - name: "check the formatting of the PR title and for missing or inconsistent labels"
+        if: github.event.pull_request.draft == false
```

```suggestion
        if: github.event.pull_request.draft == false and github.event.pull_request.base == 'master'
```
It might be good to add a check that the PR is into `master` since sometimes people open PRs into other PR branches and presumably we don't care about the title or labels for those.

---

Good point, added!